retrofit/src/main/java/retrofit2/Platform.java

```diff
@@ -32,12 +32,14 @@ static Platform get() {
   }
 
   private static Platform findPlatform() {
-    try {
-      Class.forName("android.os.Build");
-      if (Build.VERSION.SDK_INT != 0) {
-        return new Android();
+    if ("Dalvik".equals(System.getProperty("java.vm.name"))) {
```

Comment:
Will it work if we look for `android.os.Looper` instead? Seems like if that class is present then we should have Android behavior